src/components/common/RJSF/templates/ObjectFieldTemplate.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react'
+import {
+    ObjectFieldTemplatePropertyType,
+    ObjectFieldTemplateProps,
+    canExpand,
+    getTemplate,
+    getUiOptions,
+    titleId,
+} from '@rjsf/utils'
+import { FieldRowWithLabel } from '../common/FieldRow'
+
+const Field = (props: ObjectFieldTemplateProps) => {
+    const {
+        disabled,
+        formData,
+        idSchema,
+        onAddClick,
+        properties,
+        readonly,
+        registry,
+        required,
+        schema,
+        title,
+        uiSchema,
+    } = props
+    const {
+        ButtonTemplates: { AddButton },
+    } = registry.templates
+    const hasAdditionalProperties = !!schema.additionalProperties
+
+    const ActionButton = canExpand(schema, uiSchema, formData) && (
+        <AddButton
+            className="object-property-expand"
+            onClick={onAddClick(schema)}
+            disabled={disabled || readonly}
+            uiSchema={uiSchema}
+            registry={registry}
+        />
+    )
+
+    const Properties = properties.map((prop: ObjectFieldTemplatePropertyType) => prop.content)
+
+    if (hasAdditionalProperties) {
+        if (properties.length) {
+            return (
+                <>
+                    <FieldRowWithLabel
+                        label={title}
+                        required={required}
+                        showLabel
+                        id={idSchema.$id}
+                        shouldAlignCenter={false}
+                    >
+                        <div>{Properties}</div>
+                    </FieldRowWithLabel>
+                    {ActionButton}
+                </>
+            )
+        } else {
+            return (
+                <FieldRowWithLabel label={title} required={required} showLabel id={idSchema.$id}>
+                    {ActionButton}
+                </FieldRowWithLabel>
+            )
+        }
+    }
+    return (
+        <>
+            {Properties}
+            {ActionButton}
+        </>
+    )
+}
+
+export const ObjectFieldTemplate = (props: ObjectFieldTemplateProps) => {
+    const { idSchema, registry, required, schema, title, uiSchema } = props
```

Comment:
same here

Reply:
props required here for passing down to the Field component